### ISSUE TITLE
validation accuracy of imagenet??

### ISSUE THREAD

**Issue by StarrySky1213** [apache/incubator-mxnet#4787]
When I train the imagenet on 5 PCs(distributed training), I find that the validation of imagenet is about 68% which is lower than 72% the mxnet doc gives. 
Here are my params: dist_kvstore=sync, num-epoch=100 , batch-size=28 ,network = inception-bn , and other params are default.
And the following is my results:val accuracy=67.2%, train accuracy=90.6% , top5-val-accuracy=87%,top5-accuracy=98%
I want to know whether my result is right?Because it is lower than the offical guide the doc gives , I an not sure of my result?

If there is any one can give me some information？thank a lot

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
